src/api.ts

```diff
@@ -0,0 +1,21 @@
+export const requestTodos = () => {
+  return fetch('https://mate.academy/students-api/todos')
```

Comment:
Move your base api path: `https://mate.academy/students-api/` to some constant and thus you can reuse it